Geometry/TrackerCommonData/data/PhaseI/trackerParameters.xml

```diff
@@ -2,7 +2,7 @@
 <DDDefinition xmlns="http://www.cern.ch/cms/DDL" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.cern.ch/cms/DDL ../../../DetectorDescription/Schema/DDLSchema.xsd">
  <ConstantsSection  label="trackerParameters.xml" eval="true">
   <Vector name="vPars" type="numeric" nEntries="6">
-    80, 52, 0, 0, 2, 8
+    80, 52, 1, 2, 2, 8
```

Comment:
Dear Ianna,
This PR was done on top of the PR that was merged already:
https://github.com/cms-sw/cmssw/pull/17384